React/tic-tac/src/app/screens/Setting/ components/IconSelector/icons.js

```diff
@@ -0,0 +1,11 @@
+const state = {
```

Comment:
I think that the icons are not part of the state. These icons can be stored in a separate `constants` file.